Lib/test/test_importlib/extension/test_loader.py

```diff
@@ -145,6 +146,15 @@ def test_reload(self):
             importlib.reload(module)
             self.assertIs(ex_class, module.Example)
 
+    def test_reload_faliure(self):
+        '''Test that reload throws ModuleNotFounderror if reloading'''
```

Comment:
`"""..."""` supports multiline comments so you can just do:

```py
'''Test that reload throws ModuleNotFounderror if reloading
a module whose spec cannot be found'''
```

I'll let Brett or Nick comment on the actual wording of docstring.